_general/g30-functions-should-do-one-thing.md

```diff
@@ -41,4 +41,4 @@ This code would be better written as:
   }
```

Comment:
Just noticed, the `e.deliveryPay(Money pay);` is missing it's argument. Could you add that as well to this PR?

```diff
-     e.deliveryPay();
+     e.deliveryPay(pay);
```

Reply:
Much appreciated, thank you 🙂

Reply:
Fixed in [this commit](https://github.com/moderatemisbehaviour/clean-code-smells-and-heuristics/commit/ffeef140dfe469c5c76691bcf6b4bd8dc9aaf546).